planet/Objects.ocd/Animals.ocd/Bat.ocd/Script.c

```diff
@@ -379,10 +393,7 @@ private func DoSonarWave(bool agressive)
 }
 
 // Cool down effect which is removed upon first timer call.
-public func FxSonarWaveCoolDownTimer(object target, proplist effect, int time)
-{
-	return FX_Execute_Kill;
-}
+local SonarWaveCoolDown = new Effect { Timer = func(int time) { return FX_Execute_Kill; }, };
```

Comment:
Yes, it's useful for multi-line proplists to have uniform lines. By ending all lines with a comma, you can easily reorder or append entries.